Integrations/integration-ThreatExchange.yml

```diff
@@ -175,7 +179,9 @@ script:
         var data = res.obj.data;
         if (data.length === 0) {
             return {Type: entryTypes.note, Contents: res.body, ContentsFormat: formats.json,
-                HumanReadable: 'ThreatExchange does not have details about ' + url + '\n'};
+                HumanReadable: 'ThreatExchange does not have details about ' + url + '\n',
+                EntryContext: {DBotScore: [{Indicator: url, Type: 'url', Vendor: 'ThreatExchange', Score: 0}]}
+            };
         }
         var ec = {};
```

Comment:
I would use the `ec` array and insert before return so the return would be almost the same for all.